controllers/hosts.go

```diff
@@ -122,6 +122,17 @@ func pull(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Create EMQX creds if not found
+	if servercfg.GetBrokerType() == servercfg.EmqxBrokerType {
```

Comment:
move this code to host authenticate handler https://github.com/gravitl/netmaker/blob/87ab4e74db717493d0eba921beef215b8915024f/controllers/hosts.go#L415